### DISCUSSION TITLE
How can I run, in workspaces, all folder-level script?

### DISCUSSION THREAD

**Discussion by tal-rofe** [pnpm/pnpm#4756] [Q&A] [answered]
Assume I have root folder, and an another folder: `apps`.
Then I have inside 2 more folders: `backend`, `frontend`.

I also have the following: `package.json`, `./apps/frontend/package.json`, `./apps/backend/package.json`

Each has a script `test`
I want to run all 3 script once. I know I can do `pnpm -r test` but it won't trigger the root level script.
How can I do so?

Trivial way is of course doing `pnpm depcheck && pnpm -r depcheck`. Any other way?

**Comment by zkochan** [comment, marked as answer]
No, there is no other way. We have changed how this works in v7: https://github.com/pnpm/pnpm/issues/2769

We did not add an option to run recursively, including in the root.
